From ddf2b34e23476ac8e9a73b5fac6f220fce7e2be3 Mon Sep 17 00:00:00 2001
From: Marco Meyer-Conde <marco@tcu.ac.jp>
Date: Fri, 18 Jul 2025 01:30:52 +0900
Subject: [PATCH] python3-fix

---
 python/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index 888e17a..e9b3b75 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -213,6 +213,10 @@ if (${ver} STREQUAL 3)
   unset(_Python3_EXECUTABLE CACHE) # Attempt to make FindPython reenterant
   unset(_Python3_INCLUDE_DIR CACHE) # Attempt to make FindPython reenterant
   unset(_Python3_INTERPRETER_PROPERTIES CACHE) # Attempt to make FindPython reenterant
+
+  list(PREPEND CMAKE_PREFIX_PATH "$ENV{PREFIX}")
+  set(Python3_ROOT_DIR "$ENV{PREFIX}")
+
   if (${verminor} STREQUAL X)
     find_package (Python3 3.6 COMPONENTS  Interpreter Development.Module)
   else()
-- 
2.47.0

